unoconv

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
@vincentbernat  Very nice! Can you put back original shebang? Because it's not tested well to do such risky changes and there is not needs for that PR. And I will merge it!